dallinger/config.py

```diff
@@ -6,6 +6,7 @@
 import imp
 import logging
 import os
+from sets import Set
```

Comment:
I think a better way of doing this would be to follow the model for mutable defaults, so:

    def register(self, key, type_, synonyms=None, sensitive=False):
        if synonyms is None:
            synonyms = set()

The problem with `Sets.set` is that it's actually a different implementation of sets, not just a different spelling.

Your suggestion to alias it would also work, but I'd suggest that if we do it that way it should be:

    EMPTY_SET = set()
    .
    .
    .
    def register(self, key, type_, synonyms=EMPTY_SET, sensitive=False):

My preference would be the former, as it's more Pythonic, but I think either is fine.